pkg/cnab/config-adapter/adapter.go

```diff
@@ -18,7 +18,7 @@ import (
 	"github.com/cnabio/cnab-go/bundle/definition"
 )
 
-const SchemaVersion = "v1.0.0"
+const SchemaVersion = "v1.2.0"
```

Comment:
I just realized that we have two constants for the current schema version, here and in manifest.go. Since that will be very easy to get out of sync, can you consolidate that into one variable, or define them in the same file and then use a string constant to parse and create the semver?

Reply:
Maybe I'm missing something, I thought the schema version in the manifest.go file is the schema version for porter.yaml. And this schema version is for cnab?

Reply:
Sorry I mispoke. I am referring to this in the extended bundle file. We are using both a hard coded schema version in adapter, and also using the constant from cnab-go, so these will easily get out of sync. 

```go
var DefaultBundleSchemaVersion = semver.MustParse(string(bundle.GetDefaultSchemaVersion()))
```

Reply:
I see. That makes sense. I found where we store most of the cnab aliases and stored it there. Let me know what do you think?